packages/terra-slide-panel/tests/wdio/slide-panel-spec.js

```diff
@@ -103,4 +103,23 @@ Terra.describeViewports('Slide panel', ['large'], () => {
       Terra.validates.screenshot({ selector: '#root' });
     });
   });
+
+  describe('Toggle the slide panel side', () => {
+    it('Switches panel side', () => {
+      browser.url('/#/raw/tests/terra-slide-panel/slide-panel/slide-panel-side-toggle');
+      expect(browser.getText('#test-slide [aria-hidden="false"]')).to.equal('Increase Count 0');
+
+      browser.click('#focus-button');
+      browser.pause(150);
+      expect(browser.getText('#test-slide [aria-hidden="false"]')).to.equal('Increase Count 1');
+
```

Comment:
Can we add a screenshot to show both panel positions?

Reply:
[Added](https://github.com/cerner/terra-framework/pull/1071/commits/f13cab61cea812d92a6bcc163b0aa814be8ab3f5#diff-ae36c71c7db76abf8fd91f20e4e2886dR116)